servicebroker/lib/config/setting.js

```diff
@@ -14,16 +14,17 @@ module.exports = function(settingsObj) {
     }
   };
 
-  var apiServer = function(apiServerUri) {
+  var cleanupAPIServerURI = function(apiServerUri) {
```

Comment:
I just want to understand about this variable name. Does api server url variable need the word 'cleanup'?